frontend/src/js/controllers/challengeCtrl.js

```diff
@@ -343,6 +343,12 @@
             onSuccess: function(response) {
                 var details = response.data;
                 vm.phases = details;
+                // console.log(vm.phases);
+                for (var i=0; i<vm.phases.results.length; i++) {
+                    vm.phases.results[i].start_date = strftime('%b %d, %Y %H:%M:%S %Z', new Date(vm.phases.results[i].start_date));
```

Comment:
Instead of showing the full timezone name can we show the short form such as EST, IST etc.

Reply:
I was also looking for the same but didn't found anything simpler than this. I will see if I can reduce it to a short form.